### PR TITLE
Improve logging experience

### PR DESCRIPTION
**Pull request by XiaoxuHU**
As discussed with Dalibor, I changed some logging properties and add "Location" for indicating where logging is coming from

### CODE REVIEW

File: AngularApp/projects/app-service-diagnostics/src/app/shared/components/integrated-solutions-view/integrated-solutions-view.component.ts [hunk 1]
```diff
@@ -3,53 +3,52 @@ import { ActivatedRoute } from '@angular/router';
 import { TelemetryService } from 'diagnostic-data';
```
Comment (XiaoxuHU) [inline AngularApp/projects/app-service-diagnostics/src/app/shared/components/integrated-solutions-view/integrated-solutions-view.component.ts:3]: No change in this file, just formatting

File: AngularApp/projects/app-service-diagnostics/src/app/shared/components/portal-referrer-resolver/portal-referrer-resolver.component.ts [hunk 2]
```diff
@@ -53,116 +53,109 @@ export class PortalReferrerResolverComponent implements OnInit {
     let path = `resource${this._resourceService.resourceIdForRouting}`;
```
Comment (XiaoxuHU) [inline AngularApp/projects/app-service-diagnostics/src/app/shared/components/portal-referrer-resolver/portal-referrer-resolver.component.ts:53]: No change in this file, just formatting

File: AngularApp/projects/app-service-diagnostics/src/app/fabric-ui/components/fabric-search-results/fabric-search-results.component.ts [hunk 3]
```diff
@@ -137,7 +137,7 @@ export class FabricSearchResultsComponent {
     this.telemetryService.logEvent('SearchItemClicked', {
       'SearchValue': this.searchValue,
       'SelectionId': feature.id,
-      'SelectionName': feature.name,
+      'DetectorName': feature.name,
```
Comment (RobiladK) [inline AngularApp/projects/app-service-diagnostics/src/app/fabric-ui/components/fabric-search-results/fabric-search-results.component.ts:140]: Is it possible to get the Category/Id from the feature?

File: AngularApp/projects/app-service-diagnostics/src/app/home/components/collapsible-menu-item/collapsible-menu-item.component.ts [hunk 4]
```diff
@@ -42,7 +42,7 @@ export class CollapsibleMenuItemComponent implements OnInit {
       }
 
       this.telemetryService.logEvent(TelemetryEventNames.CategoryNavItemClicked,{
-        'Title':this.menuItem.label
+        'DetectorName':this.menuItem.label
```
Comment (RobiladK) [inline AngularApp/projects/app-service-diagnostics/src/app/home/components/collapsible-menu-item/collapsible-menu-item.component.ts:45]: Is it possible to get the Category/CategoryId from the menuItem object?

File: AngularApp/projects/app-service-diagnostics/src/app/home/components/home/home.component.ts [hunk 5]
```diff
@@ -347,28 +344,28 @@ export class HomeComponent implements OnInit, AfterViewInit {
...
         }
         this.globals.openFeedback = true;
     }
 
     clickQuickLink(feature: Feature) {
         this._telemetryService.logEvent(TelemetryEventNames.QuickLinkClicked, {
-            'id': feature.id,
-            'name': feature.name
+            'DetectorId': feature.id,
+            'DetectorName': feature.name
```
Comment (RobiladK) [inline AngularApp/projects/app-service-diagnostics/src/app/home/components/home/home.component.ts:361]: Is it possible to get the Category/CategoryId from the feature object?

File: AngularApp/projects/app-service-diagnostics/src/app/home/components/section-divider/section-divider.component.ts [hunk 6]
```diff
@@ -47,7 +47,7 @@ export class SectionDividerComponent implements OnInit {
 
   sectionHeaderClick() {
       this.telemetryService.logEvent(TelemetryEventNames.CategoryNavItemClicked,{
-        'Title':this.label
+        'DetectorName':this.label
```
Comment (RobiladK) [inline AngularApp/projects/app-service-diagnostics/src/app/home/components/section-divider/section-divider.component.ts:50]: Can we also log category here?

Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/app/home/components/section-divider/section-divider.component.ts]: Added CategoryName

File: AngularApp/projects/app-service-diagnostics/src/app/home/components/home/home.component.ts (hunk 5 above)
Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/app/home/components/home/home.component.ts]: Added CategoryName

File: AngularApp/projects/app-service-diagnostics/src/app/home/components/collapsible-menu-item/collapsible-menu-item.component.ts (hunk 4 above)
Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/app/home/components/collapsible-menu-item/collapsible-menu-item.component.ts]: Added CategoryName

**Review by RobiladK — APPROVED** [review]
Thank you!

**Review by RobiladK — APPROVED** [review]
Thanks

**Review by SharmaAjay19 — APPROVED** [review]

**Review by RobiladK — APPROVED** [review]

**Pull request merged**